nodes/core/cv_medianBlur.py

```diff
@@ -0,0 +1,41 @@
+import uuid
+
+import bpy
+import cv2
+from ocvl.core.node_base import OCVLNodeBase, update_node, BORDER_TYPE_ITEMS
+
+class OCVLmedianBlurNode(OCVLNodeBase):
+    n_doc = "Blurs an image using the normalized box filter."
```

Comment:
Doc is wrong check on https://docs.opencv.org/3.0-beta/modules/imgproc/doc/filtering.html?highlight=medianblu#medianblur